crates/ironrdp-server/src/encoder/mod.rs

```diff
@@ -1,5 +1,7 @@
-pub(crate) mod bitmap;
-pub(crate) mod rfx;
+#![allow(unreachable_pub)]
```

Comment:
suggestion: Can you add a comment explaining the reason for silencing this lint? In this case, we need these private items to be public for re-exporting via the bench feature.

Reply:
yes, I couldn't find a better way. Do you have another idea?

Reply:
In this case, you could provide a proxy function using the private items for you while keeping them hidden.
This is not always practical depending on what you are trying to do, but for the UpdateEncoder it works decently well.

```rust
// lib.rs
#[cfg(feature = "__bench")]
pub mod bench {
    pub mod encoder {
        pub fn update_encoder<’a>(
            bitmap: &BitmapUpdate,
            quant: rfx::Quant,
            entropy_algorithm: rfx::EntropyAlgorithm,
            tile_x: usize,
            tile_y: usize,
            buf: &'a mut [u8]
        ) -> Result<EncodedTile<'a>, RlgrError> {
            let enc = rfx::UpdateEncoder::new(&bitmap, quant, algo);
            enc.encode(0, 0, &mut buf)
        }
    }
}

// bench.rs
    let mut buf = vec![0; 64 * 64 * 3];
    c.bench_function("rfxenc", |b| {
        b.iter(|| {
            let _ = update_encoder(&bitmap, quant, algo, 0, 0, &mut buf);
        })
    });
```

Reply:
done